crates/consensus/consensus/src/lib.rs

```diff
@@ -396,7 +398,11 @@ impl ConsensusError {
     }
 }
 
+
+#[cfg(feature = "std")]
+impl std::error::Error for HeaderConsensusError {}
+
 /// `HeaderConsensusError` combines a `ConsensusError` with the `SealedHeader` it relates to.
-#[derive(thiserror_no_std::Error, Debug)]
-#[error("Consensus error: {0}, Invalid header: {1:?}")]
+#[derive(derive_more::Display, Debug)]
+#[display(fmt="Consensus error: {_0}, Invalid header: {_1:?}")]
```

Comment:
and in other places as well

```suggestion
#[display(fmt = "Consensus error: {_0}, Invalid header: {_1:?}")]
```

Reply:
Will do!

Reply:
Completed